openedx/features/edly/tests/test_middleware.py

```diff
@@ -0,0 +1,48 @@
+"""
+Unit tests for Middleware.
+"""
+from django.conf import settings
+from django.test import TestCase
+from django.test.client import Client, RequestFactory
+
+from openedx.features.edly import cookies
+from openedx.features.edly.tests.factories import EdlySubOrganizationFactory, EdlyUserFactory, SiteFactory
```

Comment:
Python's import style prefer that!
```
from openedx.features.edly.tests.factories import (
EdlySubOrganizationFactory, 
EdlyUserFactory, 
SiteFactory
)
```

Reply:
Isort is removing that since it's not violating length of the line, however, I will add this explicitly

Reply:
Don't forget to put comma with the last element `SiteFactory,`